express-get-json/server.js

```diff
@@ -0,0 +1,36 @@
+import express from 'express';
+
+const grades = {
+  12: {
+    id: 12,
+    name: 'Tim Berners-Lee',
+    course: 'HTML',
+    score: 95,
+  },
+  47: {
+    id: 47,
+    name: 'Brendan Eich',
+    course: 'JavaScript',
+    score: 100,
+  },
+  273: {
+    id: 273,
+    name: 'Forbes Lindsay',
+    course: 'JavaScript',
+    score: 92,
+  },
+};
+
+const app = express();
+
+app.listen(8080, () => {
```

Comment:
It's usually best to put this at the bottom. It doesn't need to be, but it makes more sense since all the other `app` calls are setting up the server, and once that's done you start listening.